neofetch

```diff
@@ -84,6 +84,12 @@ get_distro() {
                     *) distro="Siduction ($(lsb_release -sic))"
                 esac
 
+            elif [[ -x "/usr/bin/pveversion" ]]; then
```

Comment:
Change this to `elif type -p pveversion >/dev/null; then`. It's safer to not hardcode the path.

Reply:
:+1: